packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -491,7 +491,7 @@ class _AuthenticatorBody extends StatelessWidget {
     final stateMachineBloc = InheritedAuthBloc.of(context);
     final useAmplifyTheme = InheritedConfig.of(context).useAmplifyTheme;
     final userAppTheme = Theme.of(context);
-    final isDark = MediaQuery.platformBrightnessOf(context) == Brightness.dark;
+    final bool isDark = AmplifyTheme.of(context).isDark;
```

Comment:
This needs to stay the same - AmplifyTheme.of(context).isDark will look at the closest ThemeData ancestor and check its brightness. Here - that would be the user's theme's brightness. However, we are more interested what the platform has to say because we use this to show the dark theme or not.

Reply:
Discussed offline. Will keep as is.